test/test_uop_graph.py

```diff
@@ -171,6 +162,180 @@ def _test_vec(geps):
     xy2 = tuple(UOp(UOps.GEP, dtypes.float, (val2, ), i) for i in range(2))
     self.assertIs(_test_vec(xy1+xy2).op, UOps.VECTORIZE)
 
+  def test_gep_vec_const_fold(self):
```

Comment:
can you write this with a loop? That'd be easier to read.

Reply:
Done here and with `test_phi_vec_const_fold` and `test_gep_acc_vec_fold` as well